Tool/src/build-in-docker.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/sh
+
+TARGETPLATFORM=${TARGETPLATFORM:-linux/amd64}
+BUILD_FOLDER=${TARGETPLATFORM/\//-}
+
+if [[ -d "/workspace/build/xray-${BUILD_FOLDER}" ]]; then
+  echo "Copying prebuilt binary"
+  cp /workspace/build/xray-${BUILD_FOLDER}/xray /workspace
+else
+  echo "Building from source"
+  CGO_ENABLED=0 GOOS=linux GOARCH=$(echo $TARGETPLATFORM | cut -d'/' -f2) go build -a -ldflags '-extldflags "-static"' -o /workspace/xray /workspace/cmd/tracing
```

Comment:
Will this command also build binaries for linux/arm64 platform? Not super familiar with shell. I guess I only saw `linux/amd64`  value?

Reply:
Nah - this is mostly for local development or some other random need for a user to build a docker image from source when they don't have golang installed on their computer. So just `linux/amd64` is enough for it, and our full-blown build uses the prebuilt binaries that go crosscompiles in the Github action.